doc/data/frequency-plans.yml

```
@@ -0,0 +1,122 @@
+- id: EU_863_870
+  name: Europe 863-870 MHz (SF12 for Rx2)
+  description: Default frequency plan for Europe
+  base-frequency: 868
+  country-codes: [al, ad, ao, at, bh, be, ba, bw, bg, cg, hr, cy, cz, dk, ee, sz, fi, fr, gr, hu, is, ie, it, lv, ls, li, lt, lu, mg, mw, mt, mu, md, me, mz, na, nl, mk, ph, pl, pt, ro, ru, sa, rs, sc, sk, si, za, es, se, ch, tz, tr, ae, gb, va, zm, zw]
+  file: EU_863_870.yml
+
+- id: EU_863_870_TTN
+  name: Europe 863-870 MHz (SF9 for Rx2 - recommended)
```

```suggestion
  name: Europe 863-870 MHz (SF9 for RX2 - recommended)
```

And everywhere else. `RX1` and `RX2` are always capitalized.

---

this file is scripted over from https://github.com/TheThingsNetwork/lorawan-frequency-plans/blob/master/frequency-plans.yml, which TTS also uses. I'll submit a PR there